### PR TITLE
Fix trust level not being applied

### PR DESCRIPTION
**Pull request by ImperatorS79**
Did not test it but it should be fine I guess.

### CODE REVIEW

File: Engines/Wine/Engine/Implementation/script.js [hunk 1]
```diff
@@ -356,11 +356,12 @@ class WineEngine {
         let version = "";
         let architecture = "";
         let workingContainerDirectory = this.getContainerDirectory(this.getWorkingContainer());
+        let distribution;
```
Comment (madoar) [inline Engines/Wine/Engine/Implementation/script.js]: `let distribution;` or `let distribution = "";` (e.g. similar to `version` and `architecture`)

**Comment by plata** [comment]
Shouldn't it be "trust level" in the title?

**Comment by madoar** [comment]
Is there an easy way for us to test this?

**Comment by plata** [comment]
I think it's used by an application.

**Comment by madoar** [comment]
I don't think it is used yet. As far as I could find out it (trustLevel) seems to have been added because of #914

**Comment by plata** [comment]
I see. Then I'm not sure how to test.

**Comment by madoar** [comment]
@plata I think it is fine to merge this, we lose nothing if we do

**Comment by madoar** [comment]
@plata you decide

**Pull request merged**